config_panel.toml

```diff
@@ -0,0 +1,12 @@
+version = "1.0"
+
+[main]
+name.en = "Gitea configuration"
+    [main.general]
+    name.en = "Code indexing"
+        [main.general.indexer]
+        ask.en = "Enable code search"
+        help.en = "Indexing code can take a lot of system resources, especially when first enabled or on major Gitea upgrade"
+        type = "boolean"
+        bind = "REPO_INDEXER_ENABLED:__INSTALL_DIR__/custom/conf/app.ini"
```

Comment:
Please don't use a custom binding. The default binding with the app settings is enough when we regenerate the config.